web/client/components/data/template/jsx/Template.jsx

```diff
@@ -5,10 +5,10 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const React = require('react');
```

Comment:
mmm... I tested almost everything except this. 
I noticed it was imported only by Template-test, so I thought it was safe. 
But you make me notice that this may be a little dangerous. I'll keep require for this.

Reply:
ok, as we have seen with @mbarto we can migrate this and check possible issues in a second time, currently it is not used outside babel-stand-alone specific projects.